### PR TITLE
Support Outlook Message File Format

### PR DESCRIPTION
**Pull request by mindspawn**
Enable support for Outlook .msg format.
This change introduces support for the Outlook .msg format in open-webui, enabling users to directly drag and drop single emails, multiple emails, or entire email threads from Outlook into the interface, facilitating seamless interaction.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -133,7 +133,8 @@ RUN pip3 install uv && \
     fi; \
     chown -R $UID:$GID /app/backend/data/
 
-
+# Dependency for email message extraction
```
Comment (tjbck) [inline Dockerfile:136]: Correct me if I'm wrong but I believe this part is not required.

Comment (justinh-rahb) [reply Dockerfile:136]: Indeed, already in `requirements.txt`

Comment (mindspawn) [reply Dockerfile:136]: Wow you guys are fast :) yes that's true. Good catch. My bad.
Will fix along with another formatting issue.

**Comment by tjbck** [comment]
LGTM, Thanks!

File: Dockerfile (hunk 1 above)
Comment (justinh-rahb) [reply Dockerfile]: > Wow you guys are fast :)

monsters-inc-watching-you
😂

Comment (mindspawn) [reply Dockerfile]: ha ha you guys are crazy.. you took care of it before me :) .. I thought I was going crazy as to why there were no diffs between my fork and the dev branch :)

**Pull request merged**